package.json

```diff
@@ -87,7 +87,7 @@
     "ts-node": "^10.9.2",
     "tsconfig-paths": "4.2.0",
     "typescript": "^5.7.2",
-    "typescript-eslint": "^8.14.0"
+    "typescript-eslint": "^8.16.0"
```

Comment:
This fixes the "TypeError: Error while loading rule '@typescript-eslint/no-unused-expressions': Cannot read properties of undefined (reading 'allowShortCircuit')" error.